components/automate-cli/pkg/diagnostics/integration/helpers.go

```diff
@@ -6,7 +6,7 @@ import (
 	"net/http"
 	"time"
 
-	uuid "github.com/gofrs/uuid"
+	uuid "github.com/chef/automate/lib/uuid4"
```

Comment:
what's this needed for?

Reply:
He's just updating to use our wrapper of the UUID library (which I didn't know we had until now, nice!). We should probably be using that everywhere.